unit_test/test_spdm_callback/test_spdm_callback.c

```diff
@@ -0,0 +1,140 @@
+/**
```

Comment:
This directory should be called `unit_test/test_spdm_sample`.

Reply:
Thanks. I have changed the folder name.